src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -338,11 +338,6 @@ private PSVirtualNetworkGateway CreateVirtualNetworkGateway()
                 {
                     vnetGateway.VpnClientConfiguration.VpnClientProtocols = this.VpnClientProtocol;
                 }
-                else
```

Comment:
Is there a test that covers this scenario?  How do we know that this doesn't break current behavior?

Reply:
This is not impacting as the behavior is handled by NRP and has been for a while now.

Reply:
Yes, but we need a test to verify this.  How do we know this doesn't break somethign somewhere else?  Please proivde test coverage.

Reply:
@markcowl Tests have been updated